drivers/watchdog/sunxi-twd.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <error.h>
+#include <mmio.h>
+#include <drivers/clock.h>
+#include <drivers/timer.h>
+#include <drivers/watchdog.h>
+#include <drivers/watchdog/sunxi-twd.h>
+
+#define CLOCK_CYCLES_MHZ    24
+
+#define TWD_STATUS_REG      0x00
+#define TWD_CTRL_REG        0x10
+#define TWD_RESTART_REG     0x14
+#define TWD_INTV_VAL_REG    0x30
```

Comment:
I know this matches the docs, but `INTV_VAL` → `INTERVAL` is less redundant (the register holds a value? they all do!) and more clear.